kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -521,6 +536,22 @@ private void startGCSToBQLoadTask() {
     loadExecutor.scheduleAtFixedRate(loadRunnable, intervalSec, intervalSec, TimeUnit.SECONDS);
   }
 
+  private Map<String, String> parseTopic2TableMapConfig(String topic2TableMapString) {
```

Comment:
I think we are not using reference of this function. please remove it. If in case we are using it then its very similar to config function. try to keep it one copy.

Reply:
We are using it in BigQuerySinkConfig#getTopic2TableMap(). I tried to abstract the logic but I didn't find it useful as here we are just iterating over the string after splitting and populating a map while in validate, we have some running validations after splitting

Reply:
I am not talking about that. same function parseTopic2TableMapConfig exist in BigQuerySinkConfig and BigQuesrySinkTask both the file

Reply:
Done. Removed it.